spec/octocatalog-diff/tests/catalog-util/builddir_spec.rb

```diff
@@ -157,6 +161,41 @@
           end
         end
       end
+      context 'with --environment' do
+        context 'with logger' do
+          it 'should install a symlink to the given environment' do
+            logger = double('Logger')
```

Comment:
The pattern elsewhere in the tests is to actually construct a real logger using:

```
logger, logger_str = OctocatalogDiff::Spec.setup_logger
```

You can then also test `logger_str.string` to be matching text if you want to verify log messages.

I realize that line 127 didn't use this pattern - must have been one I missed when I went through and tried to update all of them long, long ago...  Would you mind doing it the new way? 😸